pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -69,7 +69,7 @@ func validateLowNodeUtilizationParams(params *api.StrategyParameters) error {
 func LowNodeUtilization(ctx context.Context, client clientset.Interface, strategy api.DeschedulerStrategy, nodes []*v1.Node, podEvictor *evictions.PodEvictor) {
 	// TODO: May be create a struct for the strategy as well, so that we don't have to pass along the all the params?
 	if err := validateLowNodeUtilizationParams(strategy.Params); err != nil {
-		klog.V(1).Info(err)
+		klog.V(1).InfoS("Invalid LowNodeUtilization parameters", "err", err)
```

Comment:
Just curious, is there a reason we log all these at `klog.V(1).InfoS` instead of `klog.ErrorS`?

Reply:
Good question, technically I think it should since it's an error. And the docs say `ErrorS` should be preferred anyway.

	// Info logs a non-error message with the given key/value pairs as context.
	//
	// The msg argument should be used to add some constant description to
	// the log line.  The key/value pairs can then be used to add additional
	// variable information.  The key/value pairs should alternate string
	// keys and arbitrary values.
	Info(msg string, keysAndValues ...interface{})

	// Error logs an error, with the given message and key/value pairs as context.
	// It functions similarly to calling Info with the "error" named value, but may
	// have unique behavior, and should be preferred for logging errors (see the
	// package documentations for more information).
	//
	// The msg field should be used to add context to any underlying error,
	// while the err field should be used to attach the actual error that
	// triggered this log line, if present.
	Error(err error, msg string, keysAndValues ...interface{})

Reply:
Historical one only? Error feels more intuitive here since the strategy can no longer continue executing.

Reply:
As a separete PR since there are more messages to flip: https://github.com/kubernetes-sigs/descheduler/pull/409

Reply:
Is #409 replacing this PR or just building on this for now until it merges?

Reply:
Building on top. Those changes are semantically different. Though, I am fine just merging #409, which will merge this one as well

Reply:
Well, we already have feedback for these changes here so to keep from moving discussion around too much we can merge this one, then #409. Just wasn't sure the intention

Reply:
^It's called 'stat padding'  :)